kurtosis-module/impl/participant_network/participant_network.go

```diff
@@ -261,6 +261,22 @@ func LaunchParticipantNetwork(
 		// Each CL node will be paired with exactly one EL node
 		elClientCtx := allElClientContexts[idx]
 
+		var mevBoostCtx *mev_boost.MEVBoostContext
+		if participantSpec.BuilderNetworkParams != nil {
+			mevBoostLauncher := mev_boost.MEVBoostLauncher{
+				RelayCheck: true,
+				Relays:     participantSpec.BuilderNetworkParams.Relays,
+			}
+			mevBoostServiceId := services.ServiceID("mev-boost")
```

Comment:
Two recs here:
1) this will cause an error because each iteration of the for loop will try to start a service with the same ID ("mev-boost"), so if it's expected that we have one mev-boost client per participant (EL+CL pair) then we'll want to add an index as well, for example maybe something like `mevBoostServiceId := services.ServiceID(fmt.Sprintf("%v%v", mevBoostServiceIdPrefix, idx))`
1) if we extract `"mev-boost"` to a named constant (e.g. `mevBoostServiceIdPrefix = "mev-boost-"), then we'll be able to keep all the service ID prefixes in a single spot (e.g. `clClientServiceIdPrefix` above)

Reply:
hmm I would ideally want flexibility to say per-participant if they have mev-boost or not

it seems simplest to just have 1 per participant to get started

you can advise if the configuration exposes this amount of detail today or not

Reply:
yep the configuration you have now does allow for that